google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -64,6 +64,9 @@
  * an application. However, close() needs to be called on the client object to clean up resources
  * such as threads during application shutdown.
  *
+ * <p>This client can be safely shared across multiple threads except for the Batcher instances
+ * returned from bulk operations, eg. `newBulkMutationBatcher()`, `newBulkReadRowsBatcher()`.
+ *
```

Comment:
Also, the returned `ServerStream` from  `readRows` can't be used across multiple threads

Reply:
Thanks for the review!!!

I have added similar comment on `BigtableDataClient#readRows()`. PTAL